packages/console/src/components/Switch/index.module.scss

```diff
@@ -0,0 +1,61 @@
+@use '@/scss/underscore' as _;
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 40px;
+  height: 24px;
+
+  input {
+    opacity: 0%;
+    width: 0;
+    height: 0;
+  }
+
+  .slider {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: var(--color-neutral-90);
+    transition: 0.4s;
+    border-radius: 24px;
```

Comment:
border-radius should be 16px

Reply:
updated with `12px`, half of the height which is `24px`